supabase/migrations/43_payment_emails.sql

```diff
@@ -0,0 +1,292 @@
+begin;
+
+create or replace type alert_type as enum (
+  'free_trial',
+  'free_trial_ending',
+  'free_trial_stalled',
+  'missing_payment_method',
+  'data_movement_stalled',
+  'data_not_processed_in_interval' -- Old alert type
+);
+
+-- In order to allow alerts to contain arguments after they're done firing
+-- we need to refactor alerts to contain a `firing` boolean, rather than
+-- simply omitting no-longer-firing alerts from the view.
+create type alert_snapshot as (
```

Comment:
I don't actually see this used.

If it makes sense to have views return it, that's great.
Or, I'm perfectly happy with them returning the same consistent shape as they are right now, and this could simply be removed.